libraries/AdaptiveExpressions/ExpressionFunctions.cs

```diff
@@ -2168,6 +2224,60 @@ private static (TimexProperty, string) ParseTimexProperty(object timexExpr)
             return (parsed, error);
         }
 
+        private static (CultureInfo, string) TryParseLocale(string localeStr)
+        {
+            CultureInfo result = null;
+            string error = null;
+            try
+            {
+                result = new CultureInfo(localeStr);
+            }
+            catch
```

Comment:
Consider typing this exception.

Reply:
changed to the e.message as the final error.